feature-libs/checkout/core/store/effects/checkout.effect.ts

```diff
@@ -274,13 +274,8 @@ export class CheckoutEffects {
         .set(payload.userId, payload.cartId, payload.paymentDetails.id)
         .pipe(
           map(() => {
-            let digits = payload.paymentDetails.cardNumber;
-            digits = digits.substring(digits.length - 4, digits.length);
-            this.globalMessageService?.add(
-              {
-                key: 'paymentMethods.paymentMethodSelectedSucess',
-                params: { digits: digits },
-              },
+            this.globalMessageService.add(
+              { key: 'paymentMethods.paymentMethodSelectedSuccess' },
```

Comment:
wondering why we're not doing the same as the one in develop? 

I'm in favor for not showing the last 4 digits as well, but maybe Benjamin asked Manu about if it should be displayed

Reply:
same goes with the listeners for base

Reply:
Also, this thing had created a bug in a way that when we set payment details, it will throw the error message when checkout was cleared. Exist in develop and now in out checkout branch.

Basically, when we're in b2b, and when we navigate to account, it 'clears' checkout. So when we go back to credit card, and go through the checkout steps to payment details, it will fire the global error message. 

P.S. I would say create a TODO in our list to fix later unless you want to fix it in this PR.

Reply:
> wondering why we're not doing the same as the one in develop?

I'm not comfortable manipulating the credit card info more than we need to on the FE. 
Another reason is: do we really need to specify which payment detail is selected?
In screen readers, when selecting a payment detail, as a user you'll know which one you want to select, and you just need a confirmation if it happened. To me, it seems kind of redundant to specify which payment detail was selected.

> but maybe Benjamin asked Manu about if it should be displayed

I don't see it the [ticket](https://github.com/SAP/spartacus/issues/13729). And the screenshot in the ticket clearly doesn't have any credit card digits.

> Exist in develop and now in out checkout branch

Can you provide the steps for repro? I was trying to repro it, but couldn't.

Reply:
Will post on the DM. Miguel agrees in keeping the last 4 digits.

Reply:
> Can you provide the steps for repro? I was trying to repro it, but couldn't.

1 - use credit card, and place order. and restart the checkout flow with card. You'll see the global message gets fired because it calls to set default payment.

Another case
- make sure you have address / payment set

1 - choose account and click next to go to shipping address setp
2 - you decide to not checkout as account anymore, so you go back to the payment type step, and choose card
3 - when you get to the payment step, it will throw the global message automatically

(same case for both develop and this branch)

Reply:
^ very weird as well when using express checkout. So when we get from cart page -> review-submit step, it will throw the error since under the hood, we dispatched the action to set payment details.

Reply:
Fixed express checkout.
Let's discuss the other stuff offline.